include/boost/beast/_experimental/core/ssl_stream.hpp

```diff
@@ -106,24 +105,14 @@ class ssl_stream
         boost::asio::ssl::context& ctx)
         : p_(new stream_type{
             std::forward<Arg>(arg), ctx})
-        , ctx_(&ctx)
     {
     }
 
     /// Move Constructor
-    ssl_stream(ssl_stream&& other)
```

Comment:
Since the move constructor and move assignment are implicitly generated, this could be removed.

Reply:
Note that this changes the behavior: `.get_executor()` on a moved-from stream will now be UB (null-deref). Perhaps the behavior on move of this stream should be documented? (it differs from how I/O objects in ASIO behave).

Reply:
Ehh, I just noticed `.get_executor()` already behaves like that, so no change. Sorry for the noise!